config/creatures.cfg

```diff
@@ -13,9 +13,6 @@
 #                     workers will dig and claim tiles. When idle, fighters with a digRate > 0 can dig tiles as indicated
```

Comment:
I have a problem with this commit: While this simplifies a lot of things, it isn't modder friendly. Is it at least documented in the wiki how to rescale a model for OD?